moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -71,8 +73,12 @@ class PlanningRequestAdapter
     // Get name of derived adapter
     std::string adapter_name = typeid(*this).name();
     // Try to demangle the name
-    int status;
+     int status = 1;
```

Comment:
Looks like the github web interface for editing, messed up indentation. Please check with clang-format.
```suggestion
    int status = 1;
```